app/models.py

```diff
@@ -191,13 +191,18 @@ class Template(db.Model, Versioned):
 EMAIL_PROVIDERS = [SES_PROVIDER]
 PROVIDERS = SMS_PROVIDERS + EMAIL_PROVIDERS
 
+NOTIFICATION_TYPE = ['email', 'sms', 'letter']
+
 
 class ProviderStatistics(db.Model):
     __tablename__ = 'provider_statistics'
 
     id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
     day = db.Column(db.Date, nullable=False)
-    provider = db.Column(db.Enum(*PROVIDERS, name='providers'), nullable=False)
+    provider_id = db.Column(UUID(as_uuid=True), db.ForeignKey('provider_details.id'), index=True, nullable=False)
+    provider_stats_to_provider = db.relationship(
```

Comment:
I would prefer if it would keep the existing name 'provider'.
Most of our history, marshmallow expect 'provider' and then a 'provider_id'.